src/components/user/education/education.service.ts

```diff
@@ -92,11 +83,7 @@ export class EducationService {
     }
     const changes = this.repo.getActualChanges(ed, input);
     if (result.id !== session.userId) {
-      await this.authorizationService.verifyCanEditChanges(
-        Education,
-        ed,
-        changes,
-      );
+      this.privileges.for(session, Education).verifyChanges(changes);
```

Comment:
I hate to call this out as this feature isn't used, but it sets a bad precedent. We should have a policy declaring that owners can edit educations.
Here we've hacked around the old system that didn't support owner conditions.
At least we should leave a TODO saying this is bad and should be moved into the policies.

Reply:
Going to add this as TODO and will create a ticket to the backlog to remove this pattern from all of these as this gets used in a lot of places